ksql-engine/src/main/java/io/confluent/ksql/services/SandboxedAdminClient.java

```diff
@@ -66,9 +69,12 @@
 import org.apache.kafka.clients.admin.ListConsumerGroupOffsetsResult;
 import org.apache.kafka.clients.admin.ListConsumerGroupsOptions;
 import org.apache.kafka.clients.admin.ListConsumerGroupsResult;
+import org.apache.kafka.clients.admin.ListPartitionReassignmentsOptions;
+import org.apache.kafka.clients.admin.ListPartitionReassignmentsResult;
```

Comment:
```
[ERROR] /home/****/workspace/entinc_Contributors_ksql_PR-3218/ksql-engine/src/main/java/io/confluent/ksql/services/SandboxedAdminClient.java:77: 
Wrong lexicographical order for 'org.apache.kafka.clients.admin.NewPartitionReassignment' import. Should be before 'org.apache.kafka.clients.admin.NewPartitions'. [CustomImportOrder]
```

Reply:
Thanks, I'm currently trying to compile it myself and verify it does